### DISCUSSION TITLE
Welcome to sqlite-database-integration Discussions!

### DISCUSSION THREAD

**Discussion by aristath** [WordPress/sqlite-database-integration#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nickchomey** [comment]
Thanks for this - I've been curious about this functionality for a while but haven't been sure if it's "production ready" yet.

Is there anything in particular that is incomplete, missing or otherwise a pain point?

**Reply by nickchomey** [reply]
@aristath any thoughts on my question? I remain quite curious about this, but nothing more til I have some context about what tradeoffs might come with it that I could look out for/test.
